ocfweb/docs/views/servers.py

```diff
@@ -125,30 +130,16 @@ def get_hosts():
                 if child:
                     del servers[child.hostname]
                     children.append(Host(child.hostname, 'vm', child.description, ()))
-            # Associate host with its children and move it to the hypervisors dictionary
+            # Associate host with its children and specify type
             del servers[h.hostname]
-            hypervisors[h.hostname] = Host(
+            servers[h.hostname] = Host(
                 h.hostname,
                 'hypervisor',
                 h.description,
                 children,
```

Comment:
You should specify keyword arguments here instead of relying on order. That way if the `Host` object ever changes, you won't put the wrong fields in the wrong place.